proto/one/operations/spreadsheet/computation_binding.proto

```diff
@@ -29,6 +29,9 @@ message ComputationBinding {
    // details on when to execute computation and where to put the output defaults to every row
    ExecutionDetails executionDetails = 7;
 
+   // expression text of the computation
+   string expression = 8;
```

Comment:
Is this new field meant to be required? It should be type google.ptotobuf.StringValue if it is optional.

Reply:
This should be optional and so should use the nullable wrapper as Kent mentioned.

Reply:
thanks for the suggestion. I have updated the code